packages/voice/tests/rest/v1/calls/calls.steps.ts

```diff
@@ -0,0 +1,118 @@
+import { CallsApi, VoiceService, Voice  } from '../../../../src';
+import { Given, When, Then } from '@cucumber/cucumber';
+import * as assert from 'assert';
+
+let callsApi: CallsApi;
+let callInformation: Voice.GetCallInformation;
+let updateCallResponse: void;
+let error: any;
+let manageWithCallLegResponse: void;
+
+Given('the Voice service "Calls" is available', () => {
+  const voiceService = new VoiceService({
+    applicationKey: 'appKey',
+    applicationSecret: 'appSecret',
+    voiceHostname: 'http://localhost:3019',
+  });
+  callsApi = voiceService.calls;
+});
+
+When('I send a request to get a call\'s information', async () => {
+  callInformation = await callsApi.get({
+    callId: '1ce0ffee-ca11-ca11-ca11-abcdef000003',
+  });
+});
+
+Then('the response contains the information about the call', () => {
+  assert.equal(callInformation.callId, '1ce0ffee-ca11-ca11-ca11-abcdef000003');
+  const participant: Voice.Participant = {
+    type: 'Number',
+    endpoint: '+12017777777',
+  };
+  assert.deepEqual(callInformation.to, participant);
+  assert.equal(callInformation.domain, 'pstn');
+  assert.equal(callInformation.duration, 14);
+  assert.equal(callInformation.status, 'FINAL');
+  const result: Voice.ResultEnum = 'ANSWERED';
+  assert.equal(callInformation.result, result);
```

Comment:
nit
shouldn't IDE suggest an enum value in assertion, so this explicit binding is not really adds readability?
```suggestion
  assert.equal(callInformation.result, 'ANSWERED');
```

Reply:
This is done on purpose, in order to break the transpilation in case the enum changes. The assert mechanism would not detect it otherwise.